api/apis/service_binding_handler_test.go

```diff
@@ -365,4 +366,124 @@ var _ = Describe("ServiceBindingHandler", func() {
 			})
 		})
 	})
+
+	Describe("the GET /v3/service_credential_bindings endpoint", func() {
+		makeListRequest := func(queryParams ...string) {
```

Comment:
How about renaming `makeListRequest` to `JustBeforeEach` so that the test is more idiomatic. then the function itself would not have to be explicitly invoked in all `BeforeEach`s